packages/bot-web-ui/src/components/dashboard/dashboard.scss

```diff
@@ -571,13 +606,38 @@
         position: absolute;
         top: -5rem;
         left: 20rem;
+
         @include mobile {
             top: -12rem;
             left: 10rem;
         }
     }
 }
 
+@keyframes blink {
+    0% {
+        border: 1px solid rgb(255, 68, 79);
```

Comment:
Could be a variable if we use it 2 or more times `1px solid rgb(255, 68, 79)`

Reply:
done ✅